Zend/tests/match/027.phpt

```diff
@@ -0,0 +1,92 @@
+--TEST--
+Test result of match cannot be modified by reference
+--SKIPIF--
+<?php if (!extension_loaded('Zend OPcache')) die('skip Zend OPcache extension not available'); ?>
+--INI--
+opcache.opt_debug_level=0x20000
+opcache.enable=1
+opcache.enable_cli=1
+--FILE--
+<?php
+
+// opcache can't be certain Test::usesRef is actually this method
+if (!class_exists('Test')) {
+    class Test {
+        public static function usesRef(&$x) {
+            $x = 'modified';
+        }
+        public static function usesValue($x) {
+            echo "usesValue $x\n";
+        }
+    }
+}
+
+function main() {
+    $i = 0;
+    Test::usesValue(match(true) { true => $i });
+    echo "i is $i\n";
+    $j = 1;
+    Test::usesRef(match(true) { true => $i });
+    echo "j is $j\n";
+}
+main();
+
+--EXPECTF--
```

Comment:
Do we really need an opcode dump for this test?

Reply:
Nope, I remove the opcode dump.